templates/myaccount/form-edit-password.php

```diff
@@ -41,7 +41,7 @@
 						<input type="password" class="user-registration-Input user-registration-Input--password input-text" name="password_current" id="password_current" />
 						<?php
 						if ( 'yes' === get_option( 'user_registration_login_option_hide_show_password', 'no' ) ) {
-							echo '<a href="javaScript:void(0)" class="password_preview dashicons dashicons-hidden" title=" Show password "></a>';
+							echo '<a href="javaScript:void(0)" class="password_preview dashicons dashicons-hidden" title="' . __( 'Show Password', 'user-registration' ) . '"></a>';
```

Comment:
@iamprazol Please use esc_attr__() for attribute.

Reply:
Okay !!